AdobeStockImage/etc/di.xml

```diff
@@ -27,6 +27,11 @@
                     <item name="condition" xsi:type="string">or</item>
                     <item name="field" xsi:type="string">photo</item>
                 </item>
+                <item name="premium_filter" xsi:type="array">
+                    <item name="type" xsi:type="string">premium_price_filter</item>
+                    <item name="condition" xsi:type="string">eq</item>
+                    <item name="field" xsi:type="string">all</item>
```

Comment:
I think it's worth to try to use `Magento\AdobeStockClient\Model\SearchParametersProvider\SimpleFilters` for Premium filter. It should be possible after accepting the following suggestion:
```suggestion
                    <item name="field" xsi:type="string">ALL</item>
```